### PR TITLE
u[a-s]*: normalize docs

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY

Normalize doc blocks.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

plugins/modules/udm_dns_record.py
plugins/modules/udm_dns_zone.py
plugins/modules/udm_group.py
plugins/modules/udm_share.py
plugins/modules/udm_user.py
plugins/modules/ufw.py
plugins/modules/uptimerobot.py
plugins/modules/urpmi.py
plugins/modules/usb_facts.py

### CODE REVIEW

**Comment by nate-kingsley** [comment]
👍

File: plugins/modules/udm_dns_record.py [hunk 1]
```diff
@@ -10,72 +10,69 @@
...
+DOCUMENTATION = r"""
 module: udm_dns_record
 author:
-    - Tobias Rüetschi (@keachi)
+  - Tobias Rüetschi (@keachi)
 short_description: Manage dns entries on a univention corporate server
 description:
-    - "This module allows to manage dns records on a univention corporate server (UCS).
-       It uses the python API of the UCS to create a new object or edit it."
+  - This module allows to manage dns records on a univention corporate server (UCS). It uses the python API of the UCS to create a new object
```
Comment (felixfontein) [inline plugins/modules/udm_dns_record.py:19]:
```suggestion
  - This module allows to manage dns records on a univention corporate server (UCS). It uses the Python API of the UCS to create a new object
```

File: plugins/modules/udm_dns_record.py [hunk 2]
```diff
@@ -10,72 +10,69 @@
...
+    type: str
+    required: true
+    description:
+      - Name of the record, this is also the DNS record. E.g. www for www.example.com.
+      - For PTR records this has to be the IP address.
+  zone:
+    type: str
+    required: true
+    description:
+      - Corresponding DNS zone for this record, e.g. example.com.
```
Comment (felixfontein) [inline plugins/modules/udm_dns_record.py:48]:
```suggestion
      - Corresponding DNS zone for this record, for example V(example.com).
```

File: plugins/modules/udm_dns_zone.py [hunk 3]
```diff
@@ -10,91 +10,87 @@
...
+DOCUMENTATION = r"""
 module: udm_dns_zone
 author:
-    - Tobias Rüetschi (@keachi)
+  - Tobias Rüetschi (@keachi)
 short_description: Manage dns zones on a univention corporate server
 description:
-    - "This module allows to manage dns zones on a univention corporate server (UCS).
-       It uses the python API of the UCS to create a new object or edit it."
+  - This module allows to manage dns zones on a univention corporate server (UCS). It uses the python API of the UCS to create a new object or
```
Comment (felixfontein) [inline plugins/modules/udm_dns_zone.py:19]:
```suggestion
  - This module allows to manage dns zones on a univention corporate server (UCS). It uses the Python API of the UCS to create a new object or
```

File: plugins/modules/udm_dns_zone.py [hunk 4]
```diff
@@ -10,91 +10,87 @@
...
-'''
-
-
-EXAMPLES = '''
+  state:
+    type: str
+    default: "present"
+    choices: [present, absent]
+    description:
+      - Whether the dns zone is present or not.
```
Comment (felixfontein) [inline plugins/modules/udm_dns_zone.py:34]:
```suggestion
      - Whether the DNS zone is present or not.
```

File: plugins/modules/udm_group.py [hunk 5]
```diff
@@ -10,63 +10,61 @@
...
+DOCUMENTATION = r"""
 module: udm_group
 author:
-    - Tobias Rüetschi (@keachi)
+  - Tobias Rüetschi (@keachi)
 short_description: Manage of the posix group
 description:
-    - "This module allows to manage user groups on a univention corporate server (UCS).
-       It uses the python API of the UCS to create a new object or edit it."
+  - This module allows to manage user groups on a univention corporate server (UCS). It uses the python API of the UCS to create a new object
```
Comment (felixfontein) [inline plugins/modules/udm_group.py:19]:
```suggestion
  - This module allows to manage user groups on a univention corporate server (UCS). It uses the Python API of the UCS to create a new object
```

File: plugins/modules/udm_group.py [hunk 6]
```diff
@@ -10,63 +10,61 @@
...
-'''
-
-
-EXAMPLES = '''
+      - Whether the group is present or not.
+    type: str
+  name:
+    required: true
+    description:
+      - Name of the posix group.
```
Comment (felixfontein) [inline plugins/modules/udm_group.py:39]:
```suggestion
      - Name of the POSIX group.
```

File: plugins/modules/udm_group.py [hunk 7]
```diff
@@ -10,63 +10,61 @@
...
+    type: str
+  description:
+    required: false
+    description:
+      - Group description.
+    type: str
+  position:
+    required: false
+    description:
+      - Define the whole ldap position of the group, e.g. V(cn=g123m-1A,cn=classes,cn=schueler,cn=groups,ou=schule,dc=example,dc=com).
```
Comment (felixfontein) [inline plugins/modules/udm_group.py:49]:
```suggestion
      - Define the whole LDAP position of the group, for example V(cn=g123m-1A,cn=classes,cn=schueler,cn=groups,ou=schule,dc=example,dc=com).
```

File: plugins/modules/udm_group.py [hunk 8]
```diff
@@ -10,63 +10,61 @@
...
+    type: str
+    default: ''
+  ou:
+    required: false
+    description:
+      - LDAP OU, e.g. school for LDAP OU V(ou=school,dc=example,dc=com).
```
Comment (felixfontein) [inline plugins/modules/udm_group.py:55]:
```suggestion
      - LDAP OU, for example V(school) for LDAP OU V(ou=school,dc=example,dc=com).
```

File: plugins/modules/udm_group.py [hunk 9]
```diff
@@ -10,63 +10,61 @@
...
+    type: str
+    default: ''
+  subpath:
+    required: false
+    description:
+      - Subpath inside the OU, e.g. V(cn=classes,cn=students,cn=groups).
```
Comment (felixfontein) [inline plugins/modules/udm_group.py:61]:
```suggestion
      - Subpath inside the OU, for example V(cn=classes,cn=students,cn=groups).
```

File: plugins/modules/udm_share.py [hunk 10]
```diff
@@ -10,339 +10,337 @@
...
 module: udm_share
 author:
-    - Tobias Rüetschi (@keachi)
+  - Tobias Rüetschi (@keachi)
 short_description: Manage samba shares on a univention corporate server
 description:
-    - "This module allows to manage samba shares on a univention corporate
-       server (UCS).
-       It uses the python API of the UCS to create a new object or edit it."
+  - This module allows to manage samba shares on a univention corporate server (UCS). It uses the python API of the UCS to create a new object
```
Comment (felixfontein) [inline plugins/modules/udm_share.py:19]:
```suggestion
  - This module allows to manage samba shares on a univention corporate server (UCS). It uses the Python API of the UCS to create a new object
```

File: plugins/modules/udm_user.py [hunk 11]
```diff
@@ -10,297 +10,285 @@
...
 module: udm_user
 author:
-    - Tobias Rüetschi (@keachi)
+  - Tobias Rüetschi (@keachi)
 short_description: Manage posix users on a univention corporate server
 description:
-    - "This module allows to manage posix users on a univention corporate
-       server (UCS).
-       It uses the python API of the UCS to create a new object or edit it."
+  - This module allows to manage posix users on a univention corporate server (UCS). It uses the python API of the UCS to create a new object
```
Comment (felixfontein) [inline plugins/modules/udm_user.py:19]:
```suggestion
  - This module allows to manage posix users on a univention corporate server (UCS). It uses the Python API of the UCS to create a new object
```

File: plugins/modules/udm_user.py [hunk 12]
```diff
@@ -10,297 +10,285 @@
...
+    type: str
+  groups:
+    default: []
+    description:
+      - 'POSIX groups, the LDAP DNs of the groups will be found with the LDAP filter for each group as $GROUP: V((&(objectClass=posixGroup\)(cn=$GROUP\)\)).'
+    type: list
+    elements: str
+  home_share:
+    description:
+      - Home NFS share. Must be a LDAP DN, e.g. V(cn=home,cn=shares,ou=school,dc=example,dc=com).
```
Comment (felixfontein) [inline plugins/modules/udm_user.py:112]:
```suggestion
      - Home NFS share. Must be a LDAP DN, for example V(cn=home,cn=shares,ou=school,dc=example,dc=com).
```

File: plugins/modules/ufw.py [hunk 13]
```diff
@@ -44,67 +42,60 @@
...
+      - If O(delete=true) and a value is provided for O(insert), then O(insert) is ignored.
     type: int
   insert_relative_to:
     description:
       - Allows to interpret the index in O(insert) relative to a position.
-      - V(zero) interprets the rule number as an absolute index (i.e. 1 is
-        the first rule).
-      - V(first-ipv4) interprets the rule number relative to the index of the
-        first IPv4 rule, or relative to the position where the first IPv4 rule
+      - V(zero) interprets the rule number as an absolute index (i.e. 1 is the first rule).
```
Comment (felixfontein) [inline plugins/modules/ufw.py:71]:
```suggestion
      - V(zero) interprets the rule number as an absolute index (that is, 1 is the first rule).
```

**Comment by felixfontein** [comment]
BTW, you can also batch apply suggestions :) For that you need to go to the Files Changed view (https://github.com/ansible-collections/community.general/pull/9338/files), select "add to batch" for each of them, and then apply them all at once.

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russoz thanks for your contribution!
@nate-kingsley thanks for reviewing!

**Pull request merged**